### ISSUE TITLE
FINN - MarkupSafe import Error

### ISSUE THREAD

**Issue by brsylmz23** [Xilinx/finn#588]
Hi,

When I try to reach finn notebook from my Ubuntu machine I get an error like this. Has anyone encountered it before? I can access finn workspace normally except notebook feature. 

![image](https://user-images.githubusercontent.com/41904237/162567188-65bceaef-02f8-401e-871b-869eb471fc53.png)

**Comment by AlexHw** [comment]
Have you found any solution?

**Comment by brsylmz23** [comment]
> 

Yes, I just add the following to the finn/docker/Dockerfile.finn file. I added it to line 86.
RUN pip install markupsafe==2.0.1

**Comment by SilvioRFernandes** [comment]
I tried that but the version is incompatible. So, I can resolve include in finn/docker/Dockerfile.finn file:

RUN pip install MarkupSafe==1.1.1
